src/masonite/foundation/response_handler.py

```diff
@@ -35,6 +35,8 @@ def response_handler(environ, start_response):
     to next.
     """
 
+    application.make("session").save()
```

Comment:
Is this the best place for this? Is there any other place we could put this?

Reply:
I agree it might not be the best place. It needs to happen before building response because, at this time we will loop on cookies that should be sent with response and "old flash messages stored in cookies" should have been deleted first. But maybe it can take place in a kind of session handler middleware. 
It's the same session for loading session from request : it could be placed somewhere else...